src/features/element-hiding.js

```diff
@@ -210,7 +212,7 @@ function injectStyleTag (rules) {
         }
     })
 
-    styleTagContents = styleTagContents.concat('{display:none!important;min-height:0!important;height:0!important;}')
+    styleTagContents = styleTagContents.concat('){display:none!important;min-height:0!important;height:0!important;}')
```

Comment:
This seems like it might be easily break in future. Could we create a variable for the selector specifically something like:

```suggestion
    let selector = ''
    rules.forEach((rule, i) => {
        if (i !== rules.length - 1) {
            selector = selector.concat(rule.selector, ',')
        } else {
            selector = selector.concat(rule.selector)
        }
    })
    const styleTagProperties = 'display:none!important; min-height:0!important; height:0!important;'
    const styleTagContents = `${forgivingSelector(selector)} {${styleTagProperties}}`
    injectGlobalStyles(styleTagContents)
```


```javascript
function forgivingSelector (selector) {
    return `is(${selector})`
}
```

Reply:
Good call, thanks.